gapis/server/grpc.go

```diff
@@ -87,24 +87,25 @@ func (s *grpcServer) stopIfIdle(ctx context.Context, server *grpc.Server, keepAl
 	// no activity from the client in a contiguous N chunks of time.
 	// This avoids killing the server if the machine is suspended (where the
 	// client cannot send hearbeats, and the system clock jumps forward).
-	const maxMissCount = 3
-	idleTimeout /= maxMissCount
-	misses := 0
+	const waitTime = time.Second
```

Comment:
The main argument for keeping this a multiple of the timeout, is that if your timeout is long, you won't be constantly keeping your CPU from sleeping (or forcing your thread to get scheduled) and hurting battery-life / performance for unrelated processes.

Might be reasonable to say min(60s, maxMissCount/3) that way you will still get periodic reminders if you have a huge timeout, but won't wake the CPU often.